scripts/run_validator.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 
-branch=latest
+branch=remove-hard-coded-census-date
```

Comment:
revert before merge

Reply:
Reverted the change since validator PR is merged into master now.